### DISCUSSION TITLE
[SOLVED] Unexpected linebreak in the middle of my comand line

### DISCUSSION THREAD

**Discussion by TrulsHaraldsson** [wixtoolset/issues#6633] [Questions] [answered]
SOLVED IT.
I added the InstallerPlatform property to the PropertyGroup
```
<PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|x86' ">
<InstallerPlatform>x86</InstallerPlatform>
```
I thought that was not required as wix only builds for x86?

Hi, 
I have a strange issue where I get an error message
`candle.exe : error CNDL0264: The parameter 'arch' is missing or has an invalid value [C:\SKF\Repos\Observer\AptObserver\Observer.Setup\Observer.Setup.wixproj]`

and looking in the output window I can clearly see that the arch-parameter has a line break right in the middle of it.
![image](https://user-images.githubusercontent.com/4319850/141435903-dca9c165-b52c-4a03-89a4-b9a4d641b460.png)

If I copy that part of the output I can see two spaces on each side of the parameter.
` -arch "
        x86
      " `

Have anyone seen something like that before?

FYI I'm running a msbuild command like
`MSBuild.exe .\Observer.Setup\ /p:Configuration=Release`

Thanks in advance.

**Comment by robmen** [comment, marked as answer]
Moved this to Q&A and glad you found the answer:

    <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|x86' ">
    <InstallerPlatform>x86</InstallerPlatform>
